.github/workflows/release.yaml

```diff
@@ -0,0 +1,104 @@
+name: "release"
```

Comment:
We release docker image to ghcr registry as part of release. Shall that be done in this workflow?
not necessary to be done in this PR, just want to understand desired steps.

Reply:
Yup. Will do docker-related stuff after this PR, but I can take a look.